docker-compose.yml

```
@@ -10,7 +10,8 @@ volumes:
 services:
   db:
     image: postgres:12-alpine
-    container_name: circles-db
+    restart: unless-stopped
+    container_name: "${POSTGRES_HOST}"
     env_file:
```

Would it feel suitable to replace this `env_file:` declarative in this row with an explicit `environment:` dictionary that explicitly passes only those variables from `.env`, which are actually needed? This is also more verbose, and thus readable, understandable and explicit.

```suggestion
    environment:
```